src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToCopyPPFileToOutput.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using FluentAssertions;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using System.Xml.Linq;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeHaveAPpContentFile : SdkTest
+    {
+        public GivenThatWeHaveAPpContentFile(ITestOutputHelper log) : base(log)
+        {}
+
+        [Fact]
+        public void It_copies_to_output_successfully()
+        {
+            var packageReference = GetPackageReference();
+
+            TestProject testProject = new TestProject()
+            {
+                Name = "CopyPPToOutputTest",
+                IsSdkProject = true, 
+                IsExe = true, 
+                TargetFrameworks = "netcoreapp3.0"
+            };
+            testProject.PackageReferences.Add(packageReference);
+            testProject.AdditionalProperties.Add("RestoreAdditionalProjectSources", Path.GetDirectoryName(packageReference.NupkgPath));
+            //  Use a test-specific packages folder
+            testProject.AdditionalProperties["RestorePackagesPath"] = @"$(MSBuildProjectDirectory)\..\pkg";
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject);
+
+            var restoreCommand = new RestoreCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
+            restoreCommand.Execute()
+                .Should()
+                .Pass();
+
+            var buildCommand = new BuildCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name));
+            buildCommand.Execute()
+                .Should()
+                .Pass();
+
+            var outputPath = Path.Combine(testAsset.TestRoot, testProject.Name, "bin", "Debug", "netcoreapp3.0");
```

Comment:
I'd recommend using `TestProject.TargetFrameworks` as the last argument here instead of hard-coding it.  That way if we retarget the test we only have to update one place.